index.js

```diff
@@ -1,9 +1,21 @@
 import "babel-register";
+import EventEmitter from "events";
 
 import appConfig from "./config";
-import { Product, User } from "./models";
+import DirWatcher from "./modules/dirwatcher";
+import Importer from "./modules/importer";
 
 console.log(appConfig.name);
 
-new User();
-new Product();
+const emitter = new EventEmitter();
+const watcher = new DirWatcher({ emitter });
+const myImporter = new Importer();
+
+watcher.watch("./data", 1000);
+
+emitter.on("dirwatcher:changed", fileName => {
```

Comment:
Better extract this logic to `Importer` class constructor

Reply:
@AlexBryuzgin From my point of view `Importer` should not know about `Dirwatcher` module (here it is event).

Reply:
@AlexBryuzgin If I will do this then I can't use this module to import something else

Reply:
@yamixail  but in constructor of `Importer` you can just describe the behaviour of the event, like
```javascript
constructor(emitter) {
  emitter.on("dirwatcher:changed", fileName => this.import(filename)
    .then()
    .catch()
  )
}
```

Reply:
This is side effect. Class should be responsible only for import when developer needs it. In this case I can create instance that import something from another directory without emitter at all.

Reply:
@yamixail 
`Create class Importer. It should be able to listen to DirWatcher events and start importing CSV files (converting the data to JavaScript objects) on ‘dirwatcher:changed’ event. `
So it means that this logic should be implemented in `Importer` constructor or wherever else, but not in main module
Please, fix it

Reply:
@yamixail 
`Create class Importer. It should be able to listen to DirWatcher events and start importing CSV files (converting the data to JavaScript objects) on ‘dirwatcher:changed’ event. `
So it means that this logic should be implemented in `Importer` constructor or wherever else, but not in main module
Please, fix it

Reply:
fixed